tools/generate_md_docs.py

```
@@ -1441,6 +1451,158 @@ def _extract_code_from_response(text):
+
+{chr(10).join(scenario_lines)}
+
+Generate ALL {len(scenarios)} examples now."""
+
+        # Call opencode run
+        self.log(f"One-shot prompt: {len(prompt)} chars")
+        cmd = ["opencode", "run"]
+
+        self.log(f"Running: {' '.join(cmd)}")
+        try:
+            result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)